cdap-proto/src/main/java/co/cask/cdap/proto/StreamProperties.java

```diff
@@ -15,21 +15,22 @@
  */
 package co.cask.cdap.proto;
 
+import co.cask.cdap.internal.specification.FormatSpecification;
 import com.google.common.base.Objects;
 
-import javax.annotation.Nullable;
-
 /**
  * Represents the properties of a stream.
  */
 public final class StreamProperties {
 
   private final String name;
-  private final long ttl;
+  private final Long ttl;
```

Comment:
If ttl is not given in the request, need an object here so that it can be null. Previously, the handler was working with the json directly to handle that case.